.ci/Makefile

```diff
@@ -146,10 +153,10 @@ setup-node: .runID
 		-i $(NODE_IP_ADDRESS),
 
 .PHONY: create-node
-create-node: provision-node setup-node
+create-node: provision-node setup-node validate-aws
```

Comment:
Is this needed? The two downstream goals already check for the validation

Reply:
let me remove it

Reply:
```suggestion
create-node: provision-node setup-node
```